docs/en/changes/changes.md

```diff
@@ -19,6 +19,7 @@
 * Remove physical index existing check and keep template existing check only to avoid meaningless `retry wait`
   in `no-init` mode.
 * Make sure instance list ordered in TTL processor to avoid TTL timer never runs.
+* Support monitoring PostgreSQL slow SQLs.
```

Comment:
You had a `* Update MySQL dashboard to visualize collected slow SQLs.` change log, I think you miss the one for PostgreSQL?

Reply:
yes